server/src/main/java/io/crate/analyze/ExplainAnalyzedStatement.java

```diff
@@ -43,17 +43,24 @@ public class ExplainAnalyzedStatement implements AnalyzedStatement, AnalyzedRela
     private final ProfilingContext context;
     private final List<Symbol> outputs;
     private final RelationName relationName;
+    private final boolean showCosts;
+    private static final String COLUMN_NAME = "QUERY PLAN";
```

Comment:
Output column name in psql for EXPLAIN is always "QUERY PLAN" on all options. Let's follow this.